dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/io/OutputHandle.java

```diff
@@ -48,8 +49,14 @@ public default CharSink asCharSink(@Nonnull Charset charset) throws IOException
   /**
    * Renames the temporary file to the final file.
    *
-   * <p>The stream, if any, must be closed. The temporary file must exist. Do NOT call commit() in a
-   * try-with-resources or finally block, or you risk committing bad data.
```

Comment:
What invalidated this comment?

Reply:
Dawid, did some research and concluded it is safe to delete, IIRC.
@dawidxc am I retelling my fake memories?